katacomb/katacomb/img_facade.py

```diff
@@ -426,9 +426,11 @@ def GetPlane(self, array=None, plane=[1, 1, 1, 1, 1]):
                    "from image '%s'" % (plane, self.name))
 
         try:
-            if array is None:
-                array = Image.PGetFArray(self.img)
             self.img.GetPlane(array, plane, self._err)
+            if array is None:
+                # Plane is retrieved to self.img.FArray
+                # Return a deep copy.
+                array = FArray.PCopy(self.img.FArray, self._err)
```

Comment:
I don't have the big picture of the error handling, but do you need to check whether GetPlane put an error into `self._err` before doing this step (which presumably *also* updates `self._err` with error information)?

Reply:
The way it works is that if a task (i.e. GetPlane) puts an error flag into self._err all future task calls using the same `_err` (i.e. FArray.PCopy) will do nothing.
The subsequent `handle_obit_error` call will then log the error from GetPlane once it is reached if GetPlane flagged something into self._err.